Bot/Cron/midnightCron.js

```diff
@@ -0,0 +1,48 @@
+import { DecrementStrike } from "../MessageHandlers/Strike/decrement";
+import { RandomElement } from "../MessageHandlers/Utils/random";
+import { AddRoleById } from "../MessageHandlers/Utils/roles";
+import { RemoveRoleById } from "../MessageHandlers/Utils/roles";
+import { ROLES } from "../MessageHandlers/Constants/guildData";
+
+const cron = require("node-cron");
+
+function MidnightCron(client) {
+	RunMidnightTasks(client);
+	cron.schedule("0 0 * * *", () => {
+		RunMidnightTasks(client);
+	});
+}
+
+function RunMidnightTasks(client) {
+	// The section below gets the first guild from the list
+	// This won't work if the bot is in multiple servers
+	const guildsCache = client.guilds.cache;
+	const guildId = guildsCache.keys().next().value;
```

Comment:
Could you just add a constant for the HappyFriendTime discord and get that guild from guild cache? Atm the bot is in multiple servers (my own) so not sure if it'd break?